backend/src/utils/gitScraper.js

```diff
@@ -0,0 +1,23 @@
+const rp = require('request-promise')
+const cheerio = require('cheerio')
```

Comment:
remember to add these two to your dependencies in package.json. That’s why the tests failed.